### PR TITLE
Adding github links to boxes and fixing redirects on events page

### PR DESCRIPTION
**Pull request by leeftk**
This PR addresses two different issues. 

First it adds a button to each of the cards on the boxes page that links to that specific boxes repo. #1215 

Secondly it fixes the redirects on the events page when clicking the watch now button. Before these were sending users to a page that no longer existed but now they will send users to the appropriate youtube page associated with the post. #1199

### CODE REVIEW

File: src/events/data.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "location": "Live Stream Webinar",
     "image2": "/img/live-stream3.png",
     "excerpt": "In the series finale we explore all the new chocolatey goodness in the Truffle Suite, including the latest features in Truffle Teams, the CLI, and beyond. Attendees will get hands-on with features such as the Teams Contract Manager, Debugger Enhancements, TruffleDB, and our recent FileCoin integration.",
-    "eventbrite": "https://trfl.co/truffle-webinar-series5",
+    "youtube": "https://www.youtube.com/watch?v=jvMFS6HcBIM&t=11s",
```
Comment (kevinweaver) [inline src/events/data.json:9]:
```suggestion
    "youtube": "https://www.youtube.com/watch?v=jvMFS6HcBIM",
```

File: src/events/data.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "location": "Live Stream Webinar",
     "image2": "/img/live-stream3.png",
     "excerpt": "In this episode we continue our exploration of Truffle Teams and get hands on with some of its more advanced features, including visually deploying a project to a public network. Attendees will get further hands-on exposure to Truffle Teams, particular with how it adds value to the DevOps phases of a project.",
-    "youtube": "https://trfl.co/truffle-webinar-series4",
+    "youtube": "https://www.youtube.com/watch?v=obQdKE_KDrw&t=6s",
```
Comment (kevinweaver) [inline src/events/data.json:20]:
```suggestion
    "youtube": "https://www.youtube.com/watch?v=obQdKE_KDrw",
```

**Review by kevinweaver — CHANGES_REQUESTED** [review]

File: src/events/data.json (hunk 2 above)
Comment (kevinweaver) [reply src/events/data.json:20]: I think we can strip the timestamp off the urls

File: custom_theme/boxes.html [hunk 3]
```diff
@@ -62,7 +62,10 @@ <h5 class="card-title">{{ box.displayName }}</h5>
                         <span class="badge bg-secondary">{{ tag }}</span>
                       {% endfor %}
                     </p>
-                </div>
+                    <p class="card-text">
```
Comment (kevinbluer) [inline custom_theme/boxes.html:65]: Is an opening `<div>` missing from above this line?

**Review by kevinbluer — COMMENTED** [review]
One small comment...other than that all looks good.

File: custom_theme/boxes.html (hunk 3 above)
Comment (leeftk) [reply custom_theme/boxes.html:65]: Dont think so, this element I added is part of the `card-body`

Comment (kevinbluer) [reply custom_theme/boxes.html:65]: Gotcha...if it's rendering ok then all good 😃

**Review by kevinbluer — APPROVED** [review]
👍

**Pull request merged**